scripts/build/build-utils.ps1

```diff
@@ -8,36 +8,35 @@ function Get-VsWherePath {
     return Get-ExecutablePath -name "vswhere.exe" -envVar "VSWHERE_PATH"
 }
 
-function Get-MsBuildPath([ValidateSet("14.0", "15.0")][string]$msbuildVersion) {
+function Get-MsBuildPath([ValidateSet("14.0", "15.0", "16.0", "Current")][string]$msbuildVersion) {
     if ($msbuildVersion -eq "14.0") {
         return Get-ExecutablePath -name "msbuild.exe" -envVar "MSBUILD_PATH"
     }
 
-    Write-Host "Trying to find 'msbuild.exe 15' using 'MSBUILD_PATH' environment variable"
-    $msbuild15Env = "MSBUILD_PATH"
-    $msbuild15Path = [environment]::GetEnvironmentVariable($msbuild15Env, "Process")
+    #If MSBUILD_PATH environment variable is found, the version is not checked and the value of input parameter is ignored.
+    Write-Host "Trying to find 'msbuild.exe' using 'MSBUILD_PATH' environment variable"
+    $msbuildEnv = "MSBUILD_PATH"
```

Comment:
would `$msbuildPathEnvVar` be a better name?

because just seeing `$msbuildEnv` I was thinking at "msbuild environment" (like in "MSBuild 16 environment"), whereas this is a constant name for a env variable

Reply:
applies to all places where `16.0` is used